errata_tool/connector.py

```diff
@@ -310,3 +310,37 @@ def get_paginated_data(self, api_url):
             if page_number >= PAGE_LIMIT:
                 raise RuntimeError('hit pagination timeout: %d' % page_number)
         return data
+
+    def get_filter(self, endpoint, filter_arg, **kwargs):
```

Comment:
I think "filter_arg" is always going to be the string "filter", right? Do we need this variable here?

Reply:
no it is not see '/errata' ... filter_args == 'errata_filter[filter_params]'

https://github.com/yazug/errata-tool/commit/3a2bbadfb345ed46d04dca78b4444344489b80fb#diff-1a50735487483ddcd06dc38f02e580ebR379-R384

for a couple of examples, I used, extracted from various places where searching was done.